client/ngapp/views/country-view.html

```diff
@@ -7,15 +7,39 @@
       </h1>
     </div>
 
-    <p>
-      Here's what we know about anomalies with internet connections located in <span class="capitalize">{{ countryName | lowercase }}</span>.
-    </p>
+    <ul class="toc">
```

Comment:
This is a really crude implementation of a Table of Contents that doesn't actually work because it interferes with angular's routing. I tried a couple of simple techniques and none of them were satisfactory (eg. they reloaded the data viewed on the page, instead of just scrolling).